pkg/combustion/templates/31-elemental-register.sh.tpl

```diff
@@ -20,16 +37,17 @@ After=time-sync.target
 
 [Install]
 WantedBy=multi-user.target
-Alias=elemental-system-agent.service
 
 [Service]
 Type=simple
 Restart=always
 RestartSec=5s
-StandardOutput=journal+console
-StandardError=journal+console
+StandardOutput=journal
+StandardError=journal
 Environment="CATTLE_AGENT_CONFIG=/etc/rancher/elemental/agent/config.yaml"
 ExecStartPre=/bin/sh -c "mkdir -p /var/lib/elemental/agent && cp /etc/rancher/elemental/agent/elemental_connection.json /var/lib/elemental/agent"
```

Comment:
```suggestion
# On SLE Micro /var/lib is not persistent, so we copy elemental_connection.json in ExecStartPre
ExecStartPre=/bin/sh -c "mkdir -p /var/lib/elemental/agent && cp /etc/rancher/elemental/agent/elemental_connection.json /var/lib/elemental/agent"
```

Reply:
See above. I don't think we want to put this into the systemd unit file either.